src/sage/combinat/subset.py

```diff
@@ -342,9 +342,9 @@ def cardinality(self):
             sage: Subsets(3).cardinality()
             8
 
-        TESTS::
+        TESTS:
 
-            ``__len__`` should return a Python int.
+        ``__len__`` should return a Python int.::
```

Comment:
Same comment as above: either `int. ::` or `int::` (I would opt for the latter here).

Reply:
```suggestion
        ``__len__`` should return a Python int::
```